modules/apps/object/object-test/src/testFunctional/macros/Picklist.macro

```diff
@@ -120,8 +120,6 @@ definition {
 	}
 
 	macro assertDisabledPicklistName {
```

Comment:
In the future, we should delete this macro entirely. It's one step and only an assertion so it should be done on the testcase level.